src/views/Profile.vue

```diff
@@ -0,0 +1,291 @@
+            :state="getValidState('notifications_on_authored_only')"
+          >Notifications on authored only</b-form-checkbox>
+        </b-form-group>
+        <b-form-group
+          id="checkbox-group-view-authored-requests-only"
+          label-for="checkbox-view-authored-requests-only"
+        >
+          <template slot="description">
+            <p>Only requests that were authored by you will be visible.</p>
+          </template>
+          <b-form-checkbox
+            id="checkbox-view-authored-requests-only"
+            v-model="formData.profile.view_authored_requests_only"
+            :state="getValidState('view_authored_requests_only')"
+          >View authored requests only</b-form-checkbox>
+        </b-form-group>
+        <b-form-group
+          v-if="profile.is_staff"
+          id="checkbox-group-staff-view"
+          label-for="checkbox-staff-view"
+        >
+          <b-form-checkbox
+            id="checkbox-staff-view"
+            v-model="formData.profile.staff_view"
+            :state="getValidState('staff_view')"
+          >Staff view</b-form-checkbox>
+        </b-form-group>
+        <b-button type="submit" variant="primary">Save</b-button>
+        <b-button type="reset" variant="primary">Cancel</b-button>
+        <router-link
+          :to="{name: 'passwordChange'}"
+          class="btn btn-danger pull-right"
+        >Change Password</router-link>
+      </b-form>
+    </b-col>
+    <b-col md="4">
+      <h2>API Key</h2>
+      <p>
+        The following key may be used to authenticate when using the
+        <a
+          :href="observationPortalApiUrl + '/api/'"
+        >API</a>.
+        This key should be treated like a password.
+      </p>
+      <input class="form-control" :value="apiToken" onclick="this.select()" readonly />
+      <p>
+        If you think your API token may have been comprimised (by accidentally checking it in to
+        a public source code repository, emailing it out, etc) you may revoke the token to obtain
+        a new one using the button below. WARNING: This will cause any applications that use this
+        token to stop working!
+      </p>
+      <b-button @click="performRevokeToken" variant="danger">Revoke Token</b-button>
+      <h2>API Quota</h2>
+      <p>
+        The observations API is rate limited so that it remains available to all users in the event
+        a single user makes too many requests in too rapid a succession, causing denial of service to
+        other clients.
+      </p>
+      <p>The rate limit is applied on a 24 hour rolling window. Your current quota is:</p>
+      <p>
+        <strong>{{ apiQuota.used }} out of {{ apiQuota.allowed }}</strong>
+      </p>
+      <h2>Account Removal</h2>
+      <p>You may request that your account be deleted.</p>
+      <router-link
+        :to="{name: 'accountRemovalRequest'}"
+        title="Request account removal"
+      >
+        Request account removal
```

Comment:
I think the formatting is a little inconsistent. Sometimes it looks like you have:
```
<a
     :stuff here
 > Stuff within the a </a>
```

And other times its like:
```
<a :stuff>
    Stuff in the a
</a>
```

Reply:
Yeah I used VSCode to auto-format some files, which does something different from what I normally do when I type things... I think there are also some inconsistent files in master already, so I wanted to do a separate PR just to clean up formatting.